project_euler/problem_97/sol1.py

```diff
@@ -0,0 +1,38 @@
+"""
+The first known prime found to exceed one million digits was discovered in 1999,
+and is a Mersenne prime of the form 2**6972593 − 1; it contains exactly 2,098,960
+digits. Subsequently other Mersenne primes, of the form 2**p − 1, have been found
+which contain more digits.
+However, in 2004 there was found a massive non-Mersenne prime which contains
+2,357,207 digits: (28433 * (2 ** 7830457 + 1)).
+
+Find the last ten digits of this prime number.
+"""
+
+
+def compute_digits(n: int) -> str:

```

Comment:
Hi, can you change the name of the function to `solution` and apply the same elsewhere.
Refer #2471

Also, if you're going to have a parameter for the function, add the default value such that by calling the function without any parameters, it returns the answer to the problem. In this case, I think it should be 10? If it is, it seems the answer is probably incorrect: https://github.com/nayuki/Project-Euler-solutions/blob/master/python/eulertest.py#L135
Can you confirm it by submitting to the website?: https://projecteuler.net/problem=97
Thanks in advance!

Reply:
Hi @dhruvmanila, thanks for pointing out the mistake. It was a silly mistake on my part (due to the incorrect placing of the brackets). I will correct it out and also apply the changes you have suggested. Once again thanks.

Reply:
I made the changes. I shall follow this pattern for my future PR's as well. Thanks for implementing this idea! 👍

Reply:
Awesome! Thanks for doing this 👍